images/skopeo-docker-io.yaml

```diff
@@ -210,6 +210,8 @@ docker.io:
       - ">= 2.43.0-stringlabels"
     prom/pushgateway:
       - ">= v1.0.0"
+    sonobuoy/sonobuoy:
+      - "latest"
```

Comment:
This is not a semver rule and should go into `docker.io > images`, not `docker.io > images-by-semver`

Reply:
Thanks! Updated!